onnx_xla/operator_registry.cc

```diff
@@ -41,6 +43,36 @@ namespace onnx_xla  {
     return ONNXIFI_STATUS_SUCCESS;
   }
   REGISTER_OPERATOR_TRANSLATOR(Relu, translateRelu)
+
+  //To be tested
+  //If 1 input, match output value to same op input value is matched to
+  //Otherwise do successive additions, being sure to satisfy broadcast semantics
+    //If same rank or scalar, no need to specify
+    //otherwise, use numpy matching of trailing dimensions
+  onnxStatus translateSum(const Node& n, XlaBuilder& builder, ValueOpMap& valueToOp)  {
+    if (n.inputs().size() == 1)  {
+      valueToOp[n.outputs().at(0)] = valueToOp[n.inputs().at(0)];
+      return ONNXIFI_STATUS_SUCCESS;
+    }
+    auto firstOp = valueToOp[n.inputs().at(0)];
+    for (auto i = 1; i < n.inputs().size(); ++i)  {
+      auto secondOp = valueToOp[n.inputs().at(i)];
+      auto firstNDim = ShapeUtil::Rank(builder.GetShape(firstOp).ValueOrDie());
+      auto secondNDim = ShapeUtil::Rank(builder.GetShape(secondOp).ValueOrDie());
+      std::vector<int64> broadcastDims;
+      if (firstNDim != secondNDim || firstNDim != 0 || secondNDim != 0)  {
```

Comment:
Could you add a comment saying xla allows broadcasting scalar without explicit "broadcasting dimensions"?